django_tables2/tables.py

```diff
@@ -565,6 +565,24 @@ def template(self):
     def template(self, value):
         self._template = value
 
+    def get_column_class_names(self, classes_set, bound_column):
+        """
+        Returns a set of HTML class names for cells of a bound column
+        in this tables.
+        By default it uses the class names defined in the table's attributes,
+        and additionally the bound column's name.
```

Comment:
I think it might help to mention explicityly that these class attribute will be added to both the `<th>` and the `<td> tag.